cinch/group_vars/rhel8

```diff
@@ -1,19 +1,19 @@
-ansible_python_interpreter: /usr/bin/python3
+ansible_python_interpreter: /usr/libexec/platform-python
 python_pip_package: python3-pip
 
 all_repositories:
   baseos:
     name: rhel8-baseos
-    baseurl: "{{ rhel_base }}/BaseOS/$basearch/os/"
+    baseurl: "{{ rhel_base }}/$basearch/baseos/os/"
```

Comment:
You need to tell me what your base URL is. The one I'm using has a different directory structure and as I said in https://github.com/RedHatQE/cinch/pull/242#issuecomment-456774121 I've used mine changes to successfully provision a RHEL 8 slave.

Reply:
I'll contact you by email to discuss the particulars.

Reply:
@greg-hellings can we move forward with this item? You will have to resolve the repositories issue (or add them as new ones with skip_if_unavailable=1) so I can merge and finally work on the original PR.

Reply:
I've reverted these repo path changes in the latest push.